File: 0344-reverse-string/0344-reverse-string.js
/**
 * @param {character[]} s
 * @return {void} Do not return anything, modify s in-place instead.
 */
var reverseString = function(s) {
    let n=s.length;
    let half=Math.floor(n/2);
    for(let i=0;i<half;i++){
        let temp=s[i];
        s[i]=s[n-1-i];
        s[n-1-i]=temp;
    }
};

File: 0344-reverse-string/0344-reverse-string.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        x = len(s)
        for i in range(len(s)//2):
            s[i],s[x-i-1]=s[x-i-1],s[i]